### PR TITLE
bump `@guardian/eslint-config-typescript`

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

bumps `@guardian/eslint-config-typescript`

for some reason #887 didn't like it

### CODE REVIEW

File: package-lock.json
Comment (akash1810) [inline package-lock.json:1]: Is this file needed?

Comment (sndrs) [reply package-lock.json:1]: no idea? i just bumped it in the package json and ran `npm i`?

Comment (sndrs) [reply package-lock.json:1]: was that wrong?

Comment (sndrs) [reply package-lock.json:1]: no igtnore that, sorry that's my fault rushing for stand up

**Review by akash1810 — APPROVED** [review]
👍🏽  thanks!
